src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +15,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User foundUser = userService.findByEmail(email);
+        if (foundUser == null) {
+            return false;
+        }
+        return foundUser.getPassword().equals(password);
```

Comment:
Usually when we need to return boolean we don't need if block or ternary (except complicated cases where we wish to split condition to be readable)
```suggestion
        return foundUser != null && foundUser.getPassword().equals(password);
```

Reply:
Thanks for the tip. I'll keep it in mind for the future